src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Date.cs

```diff
@@ -61,7 +61,8 @@ public static void TrimDateTimeOffset(Span<byte> buffer, out int bytesWritten)
                 int fractionEnd = 19 + numFractionDigits;
 
                 // Write fraction
-                for (int i = fractionEnd; i >= curIndex; i--)
+                // Leading zeros are written because the fraction becomes zero when it's their turn
+                for (int i = fractionEnd; i > curIndex; i--)
```

Comment:
Previously, I had noticed that the `if (fraction > 0)` branch was missing test coverage. How's the coverage now?

Reply:
It's covered now and the method is now fully covered.
(Report upload is failing for some reason.)